File: projectBlueWater/src/main/java/com/lasthopesoftware/bluewater/client/playback/file/volume/IPlaybackHandlerVolumeControllerFactory.java
package com.lasthopesoftware.bluewater.client.playback.file.volume;


import com.lasthopesoftware.bluewater.client.playback.file.PositionedPlayableFile;
import com.lasthopesoftware.bluewater.client.playback.state.volume.IVolumeManagement;

public interface IPlaybackHandlerVolumeControllerFactory {
	IVolumeManagement manageVolume(PositionedPlayableFile positionedPlayableFile, float initialHandlerVolume);
}

File: projectBlueWater/src/main/java/com/lasthopesoftware/bluewater/client/playback/file/volume/IPlaybackHandlerVolumeControllerFactory.kt
package com.lasthopesoftware.bluewater.client.playback.file.volume

import com.lasthopesoftware.bluewater.client.playback.file.PositionedPlayableFile
import com.lasthopesoftware.bluewater.client.playback.volume.IVolumeManagement

interface IPlaybackHandlerVolumeControllerFactory {
    fun manageVolume(
        positionedPlayableFile: PositionedPlayableFile?,
        initialHandlerVolume: Float
    ): IVolumeManagement?
}
